### PR TITLE
Add welsh translations for arrears 

### PR DESCRIPTION
**Pull request by Gubbsy**
## Proposed changes

Add welsh translations for arrears

### CODE REVIEW

File: config/locales/funding_application/progress_and_spend/payments/high_spend_evidence/cy.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cy:
+  progress_and_spend:
+    payments:
+      high_spend_evidence:
+        page_title: "Uwchlwytho tystiolaeth o wariant"
+        page_heading: "Uwchlwytho tystiolaeth o wariant ar gyfer %{spend_type}"
+        upload_evidence_of_spend: "Uwchlwytho tystiolaeth o wariant"
+        this_could_be: "Gallai hyn fod yn dderbynneb, anfoneb neu sgrin o dderbynneb ddigidol, i ddangos yr hyn a brynwyd gennych gyda'ch arian grant."
+        add_spend: 'Ychwanegu gwariant'
```
Comment (ptrelease) [inline config/locales/funding_application/progress_and_spend/payments/high_spend_evidence/cy.yml:9]: ""

File: config/locales/cy.yml [hunk 2]
```diff
@@ -720,4 +726,258 @@ cy:
...
+            progress_update_additional_grant_condition:
+              checkbox_selection: "Dewiswch amod grant ychwanegol i roi'r wybodaeth ddiweddaraf amdano, neu dewiswch 'Nid oes gennyf ddiweddariad ar hyn eto'"
+            date_correct:
+              blank: "Dewiswch os ydych yn disgwyl cwblhau'r prosiect erbyn y dyddiad a nodwyd"
+            has_statutory_permissions_licence:
+              blank: "Dewiswch a fyddwch yn lanlwytho caniatâd neu drwyddedau "
+            progress_update_statutory_permissions_licence:
+              blank: "Dewiswch caniatâd neu drwydded statudol newydd"
+            has_risk_update:
+              blank: 'Dewiswch a oes unrhyw ddiweddariadau risg'
```
[the remark this answers is not in the document]
Comment (Gubbsy) [reply config/locales/cy.yml]: Done

File: config/locales/funding_application/progress_and_spend/payments/high_spend_evidence/cy.yml (hunk 1 above)
Comment (Gubbsy) [reply config/locales/funding_application/progress_and_spend/payments/high_spend_evidence/cy.yml]: Done

File: config/locales/funding_application/progress_and_spend/previous_submissions/previously_submitted/cy.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+cy: 
+  progress_and_spend:
+    previous_submissions:
+      previously_submitted:
+        page_title: "Diweddariadau Blaenorol"
+        page_caption: "Diweddariad prosiect neu gael taliad"
+        page_heading: "Eich cyflwyniad %{submission_date} "
```
Comment (ptrelease) [inline config/locales/funding_application/progress_and_spend/previous_submissions/previously_submitted/cy.yml:7]: Space at end

Comment (Gubbsy) [reply config/locales/funding_application/progress_and_spend/previous_submissions/previously_submitted/cy.yml]: Done

File: config/locales/funding_application/progress_and_spend/previous_submissions/submission_summary/progress_update/cy.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+cy: 
+  progress_and_spend:
+    previous_submissions:
+      submission_summary:
+        progress_update_summary:
+          page_title: "Crynodeb diweddaru cynnydd'"
```
Comment (ptrelease) [inline config/locales/funding_application/progress_and_spend/previous_submissions/submission_summary/progress_update/cy.yml:6]: Weird.  This and the english version have that spurious apostrophe at the end.  Note to double check content.

File: config/locales/funding_application/progress_and_spend/progress_update/cash_contribution/cash_contribution_question/cy.yml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+cy:
+  progress_and_spend:
+    progress_update:
+      cash_contribution:
+        cash_contribution_question:
+          page_title: "Cyfraniad arian parod "
```
Comment (ptrelease) [inline config/locales/funding_application/progress_and_spend/progress_update/cash_contribution/cash_contribution_question/cy.yml:6]: Space at end of this and english

File: config/locales/funding_application/progress_and_spend/progress_update/cash_contribution/cash_contribution_select/cy.yml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+cy:
+  progress_and_spend:
+    progress_update:
+      cash_contribution:
+        cash_contribution_select:
+          page_title: "Cyfraniad arian parod "
```
Comment (ptrelease) [inline config/locales/funding_application/progress_and_spend/progress_update/cash_contribution/cash_contribution_select/cy.yml:6]: Space at end of this and english

File: config/locales/funding_application/progress_and_spend/previous_submissions/submission_summary/progress_update/cy.yml (hunk 4 above)
Comment (Gubbsy) [reply config/locales/funding_application/progress_and_spend/previous_submissions/submission_summary/progress_update/cy.yml:6]: Fixed

File: config/locales/funding_application/progress_and_spend/progress_update/check_your_answers/cy.yml [hunk 7]
```diff
@@ -0,0 +1,26 @@
+cy:
+  progress_and_spend:
+    progress_update:
+      check_your_answers:
+        page_title: "Gwiriwch eich atebion"
+        page_heading: "Gwiriwch eich atebion"
+        page_caption: "Dywedwch wrthym sut mae'r prosiect yn mynd"
+        lluniau: "Lluniau"
+        gweithgareddau: "Cofnodion gweithgareddau"
```
Comment (ptrelease) [inline config/locales/funding_application/progress_and_spend/progress_update/check_your_answers/cy.yml:9]: Yml keys accidentally translated on lines 8 and 9

File: config/locales/funding_application/progress_and_spend/progress_update/cash_contribution/cash_contribution_question/cy.yml (hunk 5 above)
Comment (Gubbsy) [reply config/locales/funding_application/progress_and_spend/progress_update/cash_contribution/cash_contribution_question/cy.yml:6]: Fixed

File: config/locales/funding_application/progress_and_spend/progress_update/check_your_answers/cy.yml [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+        new_staff: "Staff Newydd"
+        procured_goods: "Nwyddau, gwaith neu wasanaethau gwerth £10,000 neu fwy"
+        procured_good_evidence: "Sut ydych chi am ddweud wrthym am eich caffaeliad o £10,000 neu fwy"
+        caffaeliadau: "Ynglŷn â chaffael o £10,000 neu fwy"
```
Comment (ptrelease) [inline config/locales/funding_application/progress_and_spend/progress_update/check_your_answers/cy.yml:13]: Change key to procurements.

File: config/locales/funding_application/progress_and_spend/progress_update/check_your_answers/cy.yml (hunk 7 above)
Comment (Gubbsy) [reply config/locales/funding_application/progress_and_spend/progress_update/check_your_answers/cy.yml:9]: Fixed

File: config/locales/funding_application/progress_and_spend/progress_update/check_your_answers/cy.yml (hunk 8 above)
Comment (Gubbsy) [reply config/locales/funding_application/progress_and_spend/progress_update/check_your_answers/cy.yml:13]: Fixed

File: config/locales/funding_application/progress_and_spend/progress_update/procurement/add_procurement/cy.yml [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+  progress_and_spend:
+    progress_update:
+      procurement:
+        add_procurement:
+          page_title: "Ychwanegu Caffael"
+          page_heading: "Ychwanegu caffaeliad o £10,000 neu fwy"
+          page_caption: "Dywedwch wrthym sut mae'r prosiect yn mynd"
+          name: "Enw'r cyflenwr, contractwr neu ymgynghorydd"
+          desc: "Disgrifiad byr o'r eitem neu'r gwasanaeth"
+          purchase_date: "Dyddiad prynu neu ddyddiad y dyfarnwyd y contract i chi "
```
Comment (ptrelease) [inline config/locales/funding_application/progress_and_spend/progress_update/procurement/add_procurement/cy.yml:11]: Space here.  Might be OK - on both CY an EN

File: config/locales/funding_application/progress_and_spend/progress_update/procurement/procured_goods/cy.yml [hunk 10]
```diff
@@ -0,0 +1,12 @@
+cy:
+  progress_and_spend:
+    progress_update:
+      procurement:
+        procured_goods:
+          page_title: "Nwyddau wedi'u caffael"
+          page_heading: "Ydych chi wedi caffael unrhyw nwyddau, gwaith neu wasanaethau gwerth £10,000 neu fwy ers eich diweddariad diwethaf? "
```
Comment (ptrelease) [inline config/locales/funding_application/progress_and_spend/progress_update/procurement/procured_goods/cy.yml:7]: trailing space

File: config/locales/funding_application/progress_and_spend/start/cy.yml [hunk 11]
```diff
@@ -0,0 +1,25 @@
+cy: 
+   progress_and_spend:
+    start:
+      page_title: "Cael taliad neu roi diweddariad prosiect"
+      page_heading: "Cael taliad neu roi diweddariad prosiect "
```
Comment (ptrelease) [inline config/locales/funding_application/progress_and_spend/start/cy.yml:5]: trailing space

File: config/locales/funding_application/progress_and_spend/progress_update/procurement/procured_goods/cy.yml (hunk 10 above)
Comment (Gubbsy) [reply config/locales/funding_application/progress_and_spend/progress_update/procurement/procured_goods/cy.yml:7]: Fixed

File: config/locales/funding_application/progress_and_spend/start/cy.yml (hunk 11 above)
Comment (Gubbsy) [reply config/locales/funding_application/progress_and_spend/start/cy.yml:5]: Fixed

File: config/locales/funding_application/progress_and_spend/tasks/cy.yml [hunk 12]
```diff
@@ -0,0 +1,18 @@
+cy: 
+   progress_and_spend:
+    tasks:
+      page_title: "Rheoli eich prosiect a ddyfarnwyd"
+      page_heading: "Rheoli eich prosiect a ddyfarnwyd"
+      accept_grant: "Derbyn eich Grant"
+      get_payment: Cael taliad 
+      give_progress_update: Rhowch ddiweddariad prosiect 
+      payment_and_update: Cael taliad a rhoi diweddariad prosiect 
+      links:
```
Comment (ptrelease) [inline config/locales/funding_application/progress_and_spend/tasks/cy.yml:10]: No quotes around above three lines.

Should work actually - but inconsistent, and will break if content with apostrophe etc goes in

Comment (Gubbsy) [reply config/locales/funding_application/progress_and_spend/tasks/cy.yml:10]: Fixed

**Pull request merged**